scripts/staging/extractMiniDB.py

```diff
@@ -71,8 +71,8 @@ def rundump(dump_command):
 # and tables with data that needs to be cleaned
 # dump those with no data
 
-matviews = {'raw_adu'
```

Comment:
hah does this old "mini DB" stuff even work anymore? I doubt it :P probably just remove it in a different bug, updating it now is fine I think